config/testdata/conf.good.yml

```diff
@@ -109,7 +109,8 @@ scrape_configs:
 - job_name: service-kubernetes
 
   kubernetes_sd_configs:
-  - api_servers:
+  - kind: service_endpoints
```

Comment:
Ah, I got confused.

I don't suppose we can name this in a way to reduce the chances someone else will make the same mistake?